pkg/dataconnector/driver.go

```diff
@@ -17,13 +17,13 @@
 
 package dataconnector
 
-var logger Logger = Nologger{}
+import "github.com/rs/zerolog/log"
 
 // Add an alias to the storage, if it does not exist
 func Add(s Storage, m *DataConnector) *Error {
 	exist, err := Get(s, m.Name)
 	if err != nil {
-		logger.Error(err.Description)
+		log.Error().Msg(err.Description)
```

Comment:
```suggestion
		log.Error().Err(err).Msg("")
```

Same with all other occurs